app/src/main/java/org/simple/clinic/patient/RecentPatient.kt

```diff
@@ -32,7 +34,7 @@ data class RecentPatient(
     companion object {
 
       const val RECENT_PATIENT_QUERY = """
-        SELECT P.uuid, P.fullName, P.gender, P.dateOfBirth, P.age_value, P.age_updatedAt,
+        SELECT P.uuid, P.fullName, P.gender, P.dateOfBirth, P.age_value, P.age_updatedAt, P.createdAt patientCreatedAt,
```

Comment:
What we want is the day was registered in the system, not the day the database row was created. For this, we have another property, `recordedAt`, which is what we should be using instead.

Reply:
Ah okay, I misunderstood the implementation of it. I thought `recordedAt` was being updated every time BP or blood sugar is added, I forgot it only updates if the record date was older than the patient recorded date. Will update the implementation 👍

Reply:
Done